config/environment.js

```diff
@@ -9,7 +9,7 @@ try {
 }
 
 const {
-    A11Y_AUDIT = 'false',
+    A11Y_AUDIT = 'true',
```

Comment:
if you create a `config/local.js` and put the following in it, you won't have to worry about changing things back before committing.

```js
module.exports = {
    A11Y_AUDIT: false,
};
```

Reply:
Awesome, thanks. I was just trying to figure out the right way to do that.